docs/rules.md

```diff
@@ -0,0 +1,93 @@
+# oVirt to Kubevirt rules to import virtual machine
+
+## Actions
+
+Each validation failure will result in a predefined action being performed.
+
+The description of the actions is following:
+* **Log** - rule validation failure will result in logging that information to operator logs and the CR will be processed;
+* **Warn** - rule validation failure will result in saving that information in the CR and the CR will be processed (import will be executed);
+* **Block** - rule validation failure will result in savint that information in the CR and the CR will not be processed (import will not be executed).
+
+## Networking rules
+
+### Preconditions
+
+1. Network to be used by the imported virtual machine has been properly pre-configured in the target cluster by its administrator (host network configuration and network attachment definition).
+2. SR-IOV network interfaces are not supported at this stage.
+
+### Rules
+
+ID | Predicate | Action 
+--- | --- | ---
+1 | Source network can’t be mapped to exactly one target network. Each network’s name linked to VM’s Nics (VM.nics[].name) has corresponding network with the same name in target cluster OR mapping between these names has been provided in the migration manifest. | Block
+2 | VM.nics[].interface not in [e1000, rtl8139, virtio] | Block
+3 | Vm.nics[].on_boot == false | Log
+4 | Vm.nics[].plugged == false | Warn
+5 | VM.nics[].vnic_profile.pass_through.mode == enabled | Block
+6 | VM.nics[].vnic_profile.port_mirroring == true | Warn
+7 | VM.nics[].vnic_profile.custom_properties | Warn
+8 | VM.nics[].vnic_profile.network_filter | Warn
+9 | VM.nics[].vnic_profile.qos | Log
+
+## Storage rules
+
+### Rules
```

Comment:
Please add 
1 | VM.disk_attachments is empty | Block

Reply:
:+1: